vue-components/src/components/Message.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+	<div :class="[ 'wikit', 'wikit-Message', `wikit-Message--${type}` ]">
+		<span class="wikit-Message__content">
+			<Icon
+				class="wikit-Message__icon"
+				:type="type === 'error' ? 'error' : 'alert'"
```

Comment:
Looks like we need to add more icon types / names to the Icon component for this component

Reply:
Yes, but there is a separate task for that.

Reply:
ok :+1:

Reply:
after merging the other task we should rebase and reflect the updated values here and in storybook

Reply:
Nevermind! There is a ticket for that that I hadn't seen. https://phabricator.wikimedia.org/T265583 This will be addressed in another PR.